### PR TITLE
don't draw ST_BAR or any UI in snapshots

### LINKED ISSUE #787: Saving while fast forwarding a demo causes a segfault
I ran a git bisect and https://github.com/fabiangreffrath/woof/commit/26f676e6d3107cd51733510646190668af1bd709 seems to be the first commit where this happens

### PR DESCRIPTION
**Pull request by rfomin**
Fix #787

### CODE REVIEW

**Review by fabiangreffrath — APPROVED** [review]
> don't draw ST_BAR or any UI in snapshots

You really want this, he? Fine by me then. 😉

File: src/m_snapshot.c [hunk 1]
```diff
@@ -135,10 +131,14 @@ void M_TakeSnapshot (void)
       *p++ = s[y * (SCREENWIDTH << hires) + (WIDESCREENDELTA << hires) + x];
     }
   }
+
+  R_SetViewSize(old_screenblocks);
```
Comment (fabiangreffrath) [inline src/m_snapshot.c:135]: No `R_ExecuteSetViewSize()` here?

Comment (rfomin) [reply src/m_snapshot.c:135]: It seems to work correctly without it.

**Comment by rfomin** [comment]
> > don't draw ST_BAR or any UI in snapshots
> 
> You really want this, he? Fine by me then. 😉

Yes, I think we can try it in the next version and see if people will like it or not 😄

File: src/m_snapshot.c (hunk 1 above)
Comment (fabiangreffrath) [reply src/m_snapshot.c:135]: :shrug: OK.

**Pull request merged**